lib/dev.js

```diff
@@ -15,19 +13,12 @@ class BinaryIsInPathCheck extends DoctorCheck {
   }
 
   async diagnose () {
-    let resolvedPath;
-    try {
-      let executable = system.isWindows() ? 'where' : 'which';
-      let {stdout} = await exec(executable, [this.binary]);
-      if (stdout.match(/not found/gi)) {
-        throw new Error('Not Found');
-      }
-      resolvedPath = system.isWindows() ? stdout.split(EOL)[0] : stdout.replace(EOL, '');
-    } catch (err) {
+    const resolvedPath = await resolveExecutablePath(this.binary);
+    if (!resolvedPath) {
       return nok(`${this.binary} is MISSING in PATH!`);
```

Comment:
I think it might be useful for clients to show the actual PATH value in the error message